Sm/source tracking (#272)
salesforcecli/plugin-deploy-retrieve: messages/retrieve.metadata.md

You must run this command from within a project.

> This command uses source-tracking if support on the org, unless you set the org to not track source when it was created.

Reviewer: ```suggestion
If your org allows source tracking, then this command tracks the local and remote changes in your source. Some orgs, such as production org, never allow source tracking. You can also use the "--no-track-source" flag when you create a scratch or sandbox org to disable source tracking.
```

Use this flag to override the default API version, which is the latest version supported the CLI, with the API version in your package.xml file.

# flags.ignore-conflicts.summary

If the org uses source tracking, retrieve and save to the local filesystem even if the local files have been changed.

# flags.ignore-conflicts.description

> This has no effect on orgs that don't use source tracking.

Reviewer: ```suggestion
This flag applies only to orgs that allow source tracking. It has no effect on orgs that don't allow it, such as production orgs. 
```

# spinner.polling

Waiting for the org to respond

# error.Conflicts

> The changes that would be retrieved will overwrite

Reviewer: ```suggestion
There are changes in your local files that conflict with the org changes you're trying to retrieve.
```

# error.Conflicts.Actions

> - Re-run the command with --ignore-conflicts if you want to overwrite the remote changes

Reviewer: ```suggestion
- To overwrite the local changes, rerun this command with the --ignore-conflicts flag.
```

Reviewer: @mshanemc , okay? Your original message said to overwrite the remote changes.  But this is a retrieve, so that flag will overwrite _local_ changes, no?

> - Retrieve the changes with --ignore-conflicts if you want to overwrite the local changes

Reviewer: ```suggestion
- To overwrite the remote changes, run the "sf deploy metadata" command with the --ignore-conflicts flag.
```
